### PR TITLE
restaurant website added.

### PR DESCRIPTION
**Pull request by jugeshraghav**
### Description

This is a restaurant website with a database.
### Checklist

- [x] I've been assigned an issue related to this PR.
- [ ] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [ ] I've made a README.md file for my Project.
- [ ] The README.md file of my project contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number

#104

### CODE REVIEW

**Comment by jugeshraghav** [comment]
I have deleted the PR created before because I was facing lots of issues in it . This is a new PR related to the issue . Please have a look and suggest me changes . @chicken-biryani @harshita2216

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Add your project details to index.md file please. Also please check the screenshots, they aren't visible.

File: Restaurant-website/readme.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ It is a one pager Restaurant Website with an integrated databse.
...
-![homepage](/public/images/about.png)
+![homepage](public/images/about.png)
 
 MENU
-![homepage](/public/images/menu.png)
+![homepage](public/images/menu.png)
 
 CONTACT
-![homepage](/public/images/contact.png)
+![homepage](public/images/contact.png)
```
Comment (harshita2216) [inline Restaurant-website/readme.md:15]: These aren't still working. Try it once again please?

**Review by harshita2216 — CHANGES_REQUESTED** [review]

**Comment by jugeshraghav** [comment]
thank you @harshita2216 i have added the project in the index.md and also added the images correctly.

**Review by harshita2216 — APPROVED** [review]

**Comment by jugeshraghav** [comment]
They are working now please check

**Pull request merged**